src/main/java/org/folio/edge/sip2/parser/Parser.java

```diff
@@ -40,13 +41,15 @@
   private final Charset charset;
   private final Character delimiter;
   private final Boolean errorDetectionEnabled;
+  private final String timezone;
 
   private Parser(ParserBuilder builder) {
     charset = builder.charset == null ? DEFAULT_CHARSET : builder.charset;
     delimiter = builder.delimiter == null
         ? DEFAULT_DELIMITER : builder.delimiter;
     errorDetectionEnabled = builder.errorDetectionEnabled == null
         ? DEFAULT_ERROR_DETECTION_ENABLED : builder.errorDetectionEnabled;
+    timezone = Utils.isStringNullOrEmpty(builder.timezone) ? "Etc/UTC" : builder.timezone;
```

Comment:
You mean create the constant DEFAULT_TIMEZONE = "Etc/UTC"? 

Storing the ZoneId would get us a step closer to conversion, which ultimately is used by MessageParser.convertFieldToDateTime(). I don't think we'll gain much advantage, if any, but will have to make lots of changes in the tests

Reply:
> You mean create the constant DEFAULT_TIMEZONE = "Etc/UTC"?

Yup. Or else inline all the other "DEFAULT_*" constants.

As far as the `ZoneId` thing... If we never use the string except for `ZoneId.of(tz)`, then it might just be better to store the `ZoneId`. `ZoneId.of()` seems like a complicated operation based on a quick look at the source code.

> ... but will have to make lots of changes in the tests

Hmm.. Why would we need to change tests? Isn't this an internal implementation detail of the parser? We'd still pass in a String here for the TZ, but then store and use a `ZoneId`. Anyway, no big deal. If it is a lot of work, we can consider it in another story.